src/pages/Delivery/Delivery.styles.js

```diff
@@ -0,0 +1,7 @@
+export const styleDelivery = {
+  display: "flex",
+  flexDirection: "column",
+  margin: "3rem auto",
+  width: "1152px",
+  height: "1251px",
```

Comment:
The height shouldn't be fixed. I guess we can omit this style
```suggestion
```

Reply:
ahh ok then I should do it on vh